backend/functions/utils.py

```diff
@@ -282,13 +284,58 @@ def get_batch_translations_using_google_translate(
         return str(e)
 
 
+def get_batch_translations_using_azure_translate(
+    sentence_list,
+    source_language,
+    target_language,
+    checks_for_particular_languages=False,
+):
+
+    if checks_for_particular_languages:
+        # Checks for particular languages
+        if target_language in ["Bodo", "Maithili"]:
+            target_language = "Hindi"
+        elif target_language == "Kashmiri":
+            target_language = "Urdu"
+
+    # Change the target language to the language code
+    target_lang_code = LANG_NAME_TO_CODE_AZURE[target_language]
+    source_lang_code = LANG_NAME_TO_CODE_AZURE[source_language]
+
+    # Create a translator instance
+    translator = AzureTranslator(
+        os.environ["AZURE_TRANSLATOR_TEXT_SUBSCRIPTION_KEY"],
+        os.environ["AZURE_TRANSLATOR_TEXT_REGION"],
+        os.environ["AZURE_TRANSLATOR_TEXT_ENDPOINT"],
+    )

```

Comment:
Is it recommended to create this object each and everytime we are translating a single batch? (especially because `AzureTranslator` class does an initial API call to get list of supported_languages each time we instantiate it)

Can we somehow make this object global?

Reply:
I have declared it inside the same file and I have also removed the `get_supported_languages()` call to it because I already have a json for it in `users\utils.py`